integreat_cms/cms/models/regions/region.py

```diff
@@ -321,6 +321,22 @@ class Region(AbstractBaseModel):
         ),
     )
 
+    deepl_usage_limit = models.PositiveIntegerField(
+        default=50_000,
+        blank=True,
+        verbose_name=_("Yearly DeepL machine translation budget"),
+        help_text=_("Number of words that can be machine translated by DeepL per year"),
+    )
```

Comment:
> So if there are no custom packages, and the add-on is only bookable once (?) a checkbox "add-on booked" is sufficient and I can do away with the manual entry fields?

Yes, exactly. I think it would make sense to convert the limit to a global setting instead and only add a boolean field to the region model which tells us whether the additional package was booked.
This would also make it easier to adjust these values in the future.